frontends/bmd/src/pages/review_page.tsx

```diff
@@ -516,38 +528,46 @@ export function ReviewPage(): JSX.Element {
           }
         </VariableContentContainer>
       </Main>
-      <Sidebar
-        footer={
-          <React.Fragment>
-            <SettingsTextSize
-              userSettings={userSettings}
-              setUserSettings={setUserSettings}
-            />
-            <ElectionInfo
-              electionDefinition={electionDefinition}
-              ballotStyleId={ballotStyleId}
-              precinctSelection={singlePrecinctSelectionFor(precinctId)}
-              horizontal
-            />
-          </React.Fragment>
-        }
-      >
-        <SidebarSpacer />
-        <Prose>
-          <h2 aria-hidden>Review Votes</h2>
-          <p>Confirm your votes are correct.</p>
-          <p>
-            <LinkButton
-              large
-              primary
-              to={machineConfig.appMode.isPrint ? '/print' : '/save'}
-              id="next"
-            >
-              I’m Ready to <NoWrap>Print My Ballot</NoWrap>
-            </LinkButton>
-          </p>
-        </Prose>
-      </Sidebar>
+      {isPortrait ? (
+        <ButtonFooter>
+          {printMyBallotButton}
+          {/* <Button large onPress={() => {}}>
+            English
+          </Button> */}
```

Comment:
Should you remove this comment? Or are you leaving the "English" ones in for later work?

Reply:
Same as above comment. (No, I had thought to leave this in as example of how to add language. Thoughts?) But perhaps I should create a shared component for these two comments… or just delete it all together?